src/card-composable/types.ts

```diff
@@ -18,7 +18,7 @@ export type ComponentWithOverrides = {
 
 export type CardComposableProps = StylableGridLayout;
 
-export type CardMediaProps = {
+export type CardMediaProps = GridLayoutProps & {
```

Comment:
I think you need to do it with 
```suggestion
export type CardMediaProps = StylableGridLayout & { & {
```